pkgs/applications/editors/lapce/default.nix

```diff
@@ -84,6 +84,10 @@ rustPlatform.buildRustPackage rec {
     };
   };
 
+  patches = [
+    ./disable-updater.patch
```

Comment:
you can avoid patch, and use something like this

```
postPatch = ''
    substituteInPlace lapce-ui/Cargo.toml --replace ", \"lapce-data/updater\"" ""
  '';
```

Reply:
Or just set `buildNoDefaultFeatures = true;` then enable the required features explicitly.

Reply:
I've tried that as @wiryfuture  have noted, that didn't work

i think its because lapce is split into multiple crates and that feature is in lapce-ui crate

Reply:
> Or just set `buildNoDefaultFeatures = true;` then enable the required features explicitly.

did you read pr

Reply:
Ah sorry, came from email without the context.

Reply:
> you can avoid patch, and use something like this
> 
> ```
> postPatch = ''
>     substituteInPlace lapce-ui/Cargo.toml --replace ", \"lapce-data/updater\"" ""
>   '';
> ```

Seems to work for me, change is in 👍